website/src/pages/covid/single-variant.astro

```diff
@@ -39,38 +30,37 @@ const initialMutations = getLapisMutations(pageState.variantFilter);
 <OrganismPageLayout view={view}>
     <gs-app lapis={getLapisUrl(Organisms.covid)}>
         <div class='grid-cols-[300px_1fr] gap-x-4 sm:grid'>
-            <div>
-                <div class='bg-blue-50 px-2 py-4'>
-                    <div class='mb-2 font-semibold'>Filter dataset:</div>
-                    <LocationTimeFilter
-                        fields={view.locationFields}
-                        initialLocation={pageState.baselineFilter.location}
-                        initialDateRange={pageState.baselineFilter.dateRange}
-                        earliestDate={view.earliestDate}
-                        customDateRangeOptions={view.customDateRangeOptions}
-                        organismViewKey={organismViewKey}
-                    />
-                </div>
-                <div class='border-t-2 bg-green-50 px-2 py-4'>
-                    <div class='mb-2 font-semibold'>Search variant:</div>
-                    <VariantFilter
-                        fields={[
-                            {
-                                name: view.lineageField,
-                                label: 'Pango Lineage',
-                                initialValue: pageState.variantFilter.lineage ?? undefined,
-                                type: 'pango-lineage',
-                                rename: lineageKey,
-                            },
-                        ]}
-                        initialMutations={initialMutations}
-                        organismViewKey={organismViewKey}
-                    />
-                    <div class='my-2 font-semibold'>Or select a known variant:</div>
+            <div class='bg-gray-50'>
+                <SingleVariantPageStateSelector
+                    locationFilterConfig={{
+                        locationFields: view.locationFields,
+                        initialLocation: pageState.baselineFilter.location,
+                        placeholderText: 'Sampling location',
+                    }}
+                    daterRangeFilterConfig={{
+                        initialDateRange: pageState.baselineFilter.dateRange,
+                        dateRangeOptions: view.dateRangeOptions,
+                        earliestDate: view.earliestDate,
+                        dateColumn: view.mainDateField,
+                    }}
+                    mutationFilterConfig={{ initialMutations }}
+                    lineageFilterConfig={{
+                        initialValue: pageState.variantFilter.lineage,
+                        lapisField: view.lineageField,
+                        placeholderText: 'Pango Lineage',
```

Comment:
Isn't this actually the Nextclade pango lineage?

(We could also compute the placholder text from `view.lineageField` by using sentence case or similar)

Reply:
There is now #292, which looks at the specification of the filters in the config and #293, which looks at deriving the placeholder text from the lapis field name.